### PR TITLE
"An instance of X" phpdocs removal

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 5.0
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

This PR removes all phpdocs referring to instances of X `@param X $x A X instance` (or similar).

### CODE REVIEW

**Comment by fabpot** [comment]
fabbot errors are false positive.

**Comment by nicolas-grekas** [comment]
For 4.4 at least so that we reduce merge conflict?

**Comment by fabpot** [comment]
@nicolas-grekas I spent enough time on this. 5.0 is good enough for me.

File: src/Symfony/Component/Security/Http/HttpUtils.php [hunk 1]
```diff
@@ -144,14 +141,13 @@ public function checkRequestPath(Request $request, $path)
...
-     * @param Request $request A Request instance
-     * @param string  $path    A path (an absolute path (/foo), an absolute URL (http://...), or a route name (foo))
+     * @param string $path A path (an absolute path (/foo), an absolute URL (http://...), or a route name (foo))
      *
      * @return string An absolute URL
      *
      * @throws \LogicException
      */
-    public function generateUri($request, $path)
+    public function generateUri(Request $request, $path)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Http/HttpUtils.php]: I ported the patch in this PR on 4.4 in #32974, but this file must be updated on master only, so it's not part of that other PR.

**Comment by nicolas-grekas** [comment]
Closing for #32974

**Pull request closed without merging**